src/scripts/uploadPremadeBundles.js

```diff
@@ -84,7 +85,7 @@ async function main() {
   } else {
     try {
       if (!searchPattern) {
-        searchPattern = /^[A-Z].*-bundle.json$/;
+        searchPattern = /^[A-Z].*.json$/;
```

Comment:
There's a TODO comment above the main function that says that this is currently configured for ecqm-content-r4-2021 measure bundles. We may want to update this now that we can support MADiE bundles?

Reply:
Oh good point... this is in the `else` that looks in `ecqm-content-r4-2021`. I'll revert this.